orc8r/cloud/go/obsidian/obsidian/main.go

```diff
@@ -59,6 +59,8 @@ func main() {
 		log.Fatalf("Error creating service: %s", err)
 	}
 
+	obsidian.CombineSpecAtRuntime = srv.Config.MustGetBool(obsidian.EnableDynamicSwaggerSpecs)
```

Comment:
No reason to rename this config -- just have them both be `EnableDynamicSwaggerSpecs`